recipes/pexsi/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "pexsi" %}
+{% set version = "1.2.0" %}
+{% set build = 0 %}
+{% set mpi = mpi or "mpich" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/berkeleylab/{{ name }}/downloads/{{ name }}_v{{ version }}.tar.gz
+  sha256: 8bfad6ec6866c6a29e1cc87fb1c17a39809795e79ede98373c8ba9a3aaf820dd
+  patches:
+    - superlu_dist-cname.patch
+    - cmake-shared-lib.patch
+
+build:
+  number: {{ build }}
+  skip: True  # [win]
+  script:
+    - export CXX="$PREFIX/bin/mpicxx" CC="$PREFIX/bin/mpicc" FC="$PREFIX/bin/mpifort"
+    - cmake ${CMAKE_ARGS} -DBUILD_SHARED_LIBS=ON -DPEXSI_ENABLE_OPENMP=ON -G Ninja -B _build
+    - cmake --build _build
+    - cmake --install _build
+
+  {% set mpi_prefix = "mpi_" + mpi %}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+
+  run_exports:
+    - {{ name|lower }} * {{ mpi_prefix }}_*
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - ninja
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - {{ mpi }}
+    - parmetis
+    - superlu_dist 6.2.0
```

Comment:
Probably worth pinning in our central pinning.

Reply:
Central pinning is at 7.* or similar, I hope I can drop the explicit pin here for the next version when upstream switches to superlu_dist 7.